### ISSUE TITLE
Powertoys not appearing in the store

### ISSUE THREAD

**Issue by SmartmanApps** [MicrosoftDocs/windows-uwp#4810]
Hi,

   This document states that Powertoys is available in the Microsoft Store, but I can't find it. When I click on the link in this page, it brings up a webpage for it, which has an "Install" button, and when I click on the Install button I get the option to open up the Store, which I do, and when the Store opens it isn't there. When I do a search of the Store it also doesn't show up there - only a "GPT Powertoys", which appears to be something else altogether (costs money, not free).

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 329db15d-6ba6-3603-e8e2-e48762d25431
* Version Independent ID: 56ad7882-244e-d3d6-efe6-ec5d38ffd394
* Content: [Install PowerToys](https://learn.microsoft.com/en-us/windows/powertoys/install#installing-with-microsoft-store)
* Content Source: [hub/powertoys/install.md](https://github.com/MicrosoftDocs/windows-dev-docs/blob/docs/hub/powertoys/install.md)
* Product: **dev-environment**
* Technology: **powertoys**
* GitHub Login: @alvinashcraft
* Microsoft Alias: **aashcraft**

**Comment by alvinashcraft** [comment]
Hi @SmartmanApps. Can you try it again? Maybe there was a hiccup in the store listing. It seems to be working for me. I get a small Microsoft Store popup when I click the Install button the web store.

![image](https://github.com/MicrosoftDocs/windows-dev-docs/assets/73072/8dd3397f-8a9f-40d4-98c7-b3ab8899c8d5)

If you continue to have issues, I recommend filing a new issue on the [PowerToys product repo](https://github.com/microsoft/PowerToys/issues) because I'm pretty sure it's not a documentation issue. If they do determine that something should be updated in the docs, please circle back to this thread. I'll continue to monitor replies after the issue is closed.

**Comment by SmartmanApps** [comment]
Hi @alvinashcraft Sorry didn't reply earlier - I never got a notification that you had commented. In the meantime I wanted it installed so I did what you did - clicked on the link in the doco and it worked - but originally I just went straight to the store because I knew it was supposed to be there, but it didn't show up in any search, which is when I came back to the doco. Now that my computer knows about it it can see it regardless, though it still doesn't show up in my library.

If you have someone there who's never installed it get them to search the store and see if it shows up, cos that's the circumstances under which it wasn't showing up for me. I tried it multiple times and it never showed up. It's only when I clicked on the link in the doco that I was able to install it (but as I said, now that I've installed it it shows up in searches anyway, but not in my library).

**Comment by SmartmanApps** [comment]
Hey @alvinashcraft Just letting you know I just ran into the EXACT same thing when installing WSL/Ubuntu. I'm following a blog about installing WSA, and it included a link to install Ubuntu from the Windows store. When I clicked on that link it took me to the web version, same as the link for Powertoys in the doco did, but when I look for that same package directly in the Store it doesn't show up, same as I ran into before with Powertoys. See screenshots. Some of these developer tools are having visibility issues within the on-device version of the Store (and if I click on "Open Store app" then it opens the Store but NOT at the package required - same thing happened earlier when I tried with with Powertoys (which is why I was then searching the Store for Powertoys, but not seeing it). i.e. these installs are only working if you have a link for the Web Store in the first place.
![image](https://github.com/MicrosoftDocs/windows-dev-docs/assets/20635275/c6f3e517-a376-42ce-97f0-bd537be89b22)
![image](https://github.com/MicrosoftDocs/windows-dev-docs/assets/20635275/a53c747a-e390-49f7-8853-5ac4b337fd2f)
